### PR TITLE
8.transaction-details

### PR DESCRIPTION
**Pull request by geigerzaehler242**
adding the script to print transaction details for each transaction the wallet has completed

### CODE REVIEW

File: swift/8.transaction-details.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    try wallet.sync(blockchain: blockchain, progress: nil)
+
+    let balance = try wallet.getBalance().total
+    print("Wallet descriptor is \(descriptor.asStringPrivate())")
+    print("The wallet balance is \(balance)\n")
+
+    let txList: [TransactionDetails] = try wallet.listTransactions()
+    txList.forEach { item in
+        print(item)
+        print("\n")
```
Comment (reez) [inline swift/8.transaction-details.swift:27]: The suggestion I'd give here is to combine the two print statements into one:

Maybe with something like this or anything that combines the two into one `print`...
```suggestion
        print("\(item) \n")
```

**Review by reez — CHANGES_REQUESTED** [review]
One requested change, otherwise looks great!

File: swift/8.transaction-details.swift (hunk 1 above)
Comment (geigerzaehler242) [reply swift/8.transaction-details.swift]: done

**Review by reez — APPROVED** [review]
Approved.

Reviewed, and ran locally.

**Pull request merged**